cpp/oneapi/dal/spmd/communicator.hpp

```diff
@@ -436,64 +436,64 @@ class communicator : public base {
     /// @param source_rank          The rank to receive data from.
     ///
     /// @return The object to track the progress of the operation
-    request send_receive_replace(byte_t* buf,
-                                 std::int64_t count,
-                                 const data_type& dtype,
-                                 std::int64_t destination_rank,
-                                 std::int64_t source_rank) const {
+    request sendrecv_replace(byte_t* buf,
```

Comment:
Can you please change the comment above to use the word "Shuffles" instead of "Shuffle"? To be aligned with other comments.